src/aiida/cmdline/params/options/main.py

```diff
@@ -344,23 +343,17 @@ def set_log_level(_ctx, _param, value):
     help='The format of the archive file.',
 )
 
-NON_INTERACTIVE = OverridableOption(
-    '-n',
-    '--non-interactive',
-    is_flag=True,
-    is_eager=True,
-    help='In non-interactive mode, the CLI never prompts but simply uses default values for options that define one.',
-)
-
```

Comment:
Hmm, that wasn't quite the change I was aiming for 😅 I see the point of wanting to avoid the double negative though. Ok, fine to keep the change, but I think we cannot just remove the `NON_INTERACTIVE` option as it is probably used elsewhere. So please reinstate it and just define if as a `INTERACTIVE.clone()`. Ideally we would deprecate it, but that will take some doing. We probably would have to add a `deprecated` argument to `OverridableOption` that is then checked upon option call, or the value parsing. We cannot just do it on import because then it will always show, even if the user is not actually using it

Reply:
Oh right, it can also be used _outside_ of `aiida-core`. 🤦 I'll reinstate it for now, we can think of the deprecation path later.

Reply:
Van uitstel komt afstel.....

Reply:
Can we just redefine it using `INTERACTIVE.clone()` though? As I understand it, we can't override the option name, and this would be used in the function definition of the `click` command. So I think we need to fully preserve the `NON_INTERACTIVE` option if we want to maintain backwards compatibility.

I've added the deprecation warning as requested to avoid afstel. ^^

Reply:
So now we are exactly in the situation that I was trying to avoid. We now have two options to do the same thing that need to be maintained and the old one is deprecated. Is all of that faf really necessary just to avoid a double negative in one place? It was working fine before wasn't it?

Reply:
Yeah, I hadn't considered that the options were part of the public API. I still think having an `INTERACTIVE` switch option makes more sense, but it isn't worth the deprecation. I'll see about turning the `NON_INTERACTIVE` option into a proper switch instead.